src/translib/transformer/xfmr_vrf.go

```diff
@@ -60,6 +61,8 @@ func getInternalNwInstName (name string) (string, error) {
                 return "mgmt", err
         } else if (strings.HasPrefix(name, "Vrf") == true) {
                 return "Vrf", err
+        } else if (strings.HasPrefix(name, "Vlan") == true) {
```

Comment:
Under oc-netinst:network-instance/oc-netinst:config, we have the following field transformers, which need relevant changes wrt to Vlans.
network_instance_name_field_xfmr
network_instance_type_field_xfmr
network_instance_enabled_field_xfmr
network_instance_mtu_field_xfmr